### PR TITLE
Fix windows dev env

### PR DESCRIPTION
**Pull request by nevalla**
Signed-off-by: Lauri Nevala <lauri.nevala@gmail.com>

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -20,8 +20,12 @@ compile-dev:
 	yarn compile:renderer --cache
 
 dev:
+ifeq "$(DETECTED_OS)" "Windows"
+	make init
```
Comment (nevalla) [inline Makefile:24]: TODO: `test` is not valid command in Windows and need to figure out proper way to do this on Windows

Comment (ocdi) [reply Makefile:24]: I second this PR - I had made very similar changes on my fork because I am starting to hack on this project. :-)

**Comment by ocdi** [comment]
I second this PR - I had made very similar changes on my fork because I am starting to hack on this project. :-)

File: package.json [hunk 2]
```diff
@@ -12,9 +12,9 @@
   },
   "scripts": {
     "dev": "concurrently -k \"yarn dev-run -C\" \"yarn dev:main\" \"yarn dev:renderer\"",
-    "dev-run": "nodemon --watch static/build/main.js --exec \"electron --inspect .\" $@",
-    "dev:main": "env DEBUG=true yarn compile:main --watch $@",
-    "dev:renderer": "env DEBUG=true yarn compile:renderer --watch $@",
+    "dev-run": "nodemon --watch static/build/main.js --exec \"electron --inspect .\"",
```
Comment (ocdi) [inline package.json]:
```suggestion
    "dev-run": "cross-env DEBUG=true nodemon --watch static/build/main.js --exec \"electron --inspect .\"",
```

File: package.json [hunk 3]
```diff
@@ -12,9 +12,9 @@
...
+    "dev:main": "yarn compile:main --watch",
```
Comment (ocdi) [inline package.json]:
```suggestion
    "dev:main": "cross-env DEBUG=true yarn compile:main --watch",
```

File: package.json [hunk 4]
```diff
@@ -12,9 +12,9 @@
...
+    "dev:renderer": "yarn compile:renderer --watch",
```
Comment (ocdi) [inline package.json]:
```suggestion
    "dev:renderer": "cross-env DEBUG=true yarn compile:renderer --watch",
```

**Review by ocdi — CHANGES_REQUESTED** [review]
Note the `cross-env` package will need to be added to `devDependencies`, I would add that as a suggestion but I don't think this UI allows for it :-)

**Comment by nevalla** [comment]
Thanks @ocdi 🙏 

Btw, do you have any ideas what could be Windows equivalent for this line?
```
test -f out/main.js || make init
```

So test if `out/main.js` file exists and if not, then run `make init` ?

**Comment by jakolehm** [comment]
> Btw, do you have any ideas what could be Windows equivalent for this line?
> 
> ```
> test -f out/main.js || make init
> ```
> 
> So test if `out/main.js` file exists and if not, then run `make init` ?

Maybe this helps? http://www.humbug.in/2012/makefile-check-if-a-file-exists-using-wildcard-function/

**Comment by nevalla** [comment]
> 
> Maybe this helps? http://www.humbug.in/2012/makefile-check-if-a-file-exists-using-wildcard-function/

Yes it does! Thanks!

**Review by jakolehm — APPROVED** [review]
LGTM

**Pull request merged**